### ISSUE TITLE
Actually we can use this repo as a experimental field

### ISSUE THREAD

**Issue by BreakingLead** [crablang/crab#119]
We can add some functionality to something shouldn't exist in rust, such as a way to disable orphan rule or something

**Comment by trvswgnr** [comment]
absolutely! there's a "stable" branch that will be more up to date with upstream, but the primary branch is open for experimentation. i've been syncing it with upstream since there haven't been any breaking changes yet but once there are it'll be a hard fork.
